<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInterface</class>
 <widget class="QWidget" name="TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TInterface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Number type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="_doubleType">
          <property name="text">
           <string>Double</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_complexType">
          <property name="text">
           <string>Complex</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Polynom</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="_labelReal">
            <property name="text">
             <string>Real</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="_labelImg">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="_labelA">
            <property name="text">
             <string>a</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditAR"/>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditAI"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="_labelB">
            <property name="text">
             <string>b</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditBR"/>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditBI"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="_labelC">
            <property name="text">
             <string>c</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditCR"/>
          </item>
          <item>
           <widget class="QLineEdit" name="_lineEditCI"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_labelResult">
       <property name="text">
        <string>Result.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="_calc">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_show">
         <property name="text">
          <string>Show polynom</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
